internal/pkg/vaultworker/policyToken.go

```diff
@@ -294,9 +289,9 @@ func hashFile(policyFilePtr *string, debug bool) (hashSum []byte, err error) {
 	return hashSum, err
 }
 
-// ----------------------------------------------------------
-func fatalIfErr(err error, msg string) {
+/*FatalIfErr =  Prinf() followed by a call to os.Exit(1)*/
+func FatalIfErr(err error, msg string) {
```

Comment:
this function duplicates the one in the pkisetup package

Reply:
such helper function need to be removed. will address in the next PR.

Reply:
do you have an issue to track this cleanup?

Reply:
The implementation is using default log package which needs to be changed to be consistent with edgex log mechanism. will be tracked under #73